terraform/accounts/gov/prd-plat/main.tf

```diff
@@ -0,0 +1,5 @@
+module "config_recorder" {
```

Comment:
I'm not sure what you mean about reusing the rules we have in each partition. Do you mean in terms of how we give access to deploy to each account?

Or do you just mean in terms of code structure? If so, then yeah, my thought is to mimic what we're doing in `aws-admin` where there is a separate directory for each account we deploy to. So in this case we will end up creating a directory for all of our Govcloud accounts since we want to enable AWS Config in all of them